README.md

```diff
@@ -6,12 +6,16 @@ Read our whitepaper: [Chainweb: A Proof-of-Work Parallel-Chain Architecture for
 
 ### Building with Nix
```

Comment:
I do question this. 
- Our readme is basically a Nix advertisement
- it's a novel that noobs will absolutely mess up

Reply:
Also, maybe not for this PR, we should mention binaries right about now. I'm going to look at throwing something up now.

Reply:
Yeah I'm also not sure about saying *Do Nix* at the very top.